src/main/java/logger/MessageLogger.java

```diff
@@ -0,0 +1,13 @@
+package logger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.Objects;
+
+public class MessageLogger {
+    private final Logger logger = LoggerFactory.getLogger(MessageLogger.class);
```

Comment:
so how did this help with testing if you still keep replacing a global object with mock in tests? :D

Reply:
will create a new issue and fix it there.

Reply:
what i wanted to see is

```
public class MessageLogger {
    private final Logger logger;
    public MessageLogger() {
        this(LoggerFactory.getLogger(MessageLogger.class));
    }
    MessageLogger(Logger logger) {
        this.logger = logger;
    }
    public void LogToStringToConsole(String message) {
        Objects.requireNonNull(message, "string must not be null");
        logger.info(message);
    }
}
```
and then in tests:

```
@Mock
private Logger logger;

...

final MessageLogger underTest = new MessageLogger(logger);
```